test/arcade/potts/agent/cell/PottsCellTest.java

```diff
@@ -180,7 +201,7 @@ public void getPop_defaultConstructor_returnsValue() {
     @Test
     public void getPop_valueAssigned_returnsValue() {
         int pop = randomIntBetween(0, 100);
-        PottsCell cell = new PottsCell(cellID, cellParent, pop, cellState, cellAge, cellDivisions,
+        PottsCellMock cell = new PottsCellMock(cellID, cellParent, pop, cellState, cellAge, cellDivisions,
```

Comment:
Use `PottsCell cell = new PottsCellMock(...` instead of `PottsCellMock cell = new PottsCellMock(...`